src/app/checkout/navigateToOrderConfirmation.tsx

```diff
@@ -1,7 +1,14 @@
 import { noop } from 'lodash';
 
-export default function navigateToOrderConfirmation(location = window.location): Promise<never> {
-    const url = `${location.pathname.replace(/\/$/, '')}/order-confirmation`;
+export default function navigateToOrderConfirmation(location = window.location, orderId?: number): Promise<never> {
+    let url: string;
+    const checkoutId = window.location.pathname.replace('/checkout/', '').replace('embedded-checkout', '');
+
+    if (orderId && checkoutId.length > 0) {
+        url = `/checkout/order-confirmation/${orderId.toString()}`;
```

Comment:
You can use the same `${location.pathname.replace(/\/$/, '')}/order-confirmation/${orderId.toString()}` here as well?

Reply:
I was told we don't support embedded checkout to use Buy Now Cart, hence I made the url in this way.